src/dut.rs

```diff
@@ -649,18 +710,26 @@ impl SshInfo {
     }
     // Start SSH port forwarding on a given port.
     // The future will be resolved once the first connection attempt is succeeded.
-    pub async fn start_ssh_forwarding(&self, port: u16) -> Result<async_process::Child> {
-        self.start_ssh_forwarding_in_range(Range {
-            start: port,
-            end: port + 1,
-        })
+    pub async fn start_ssh_forwarding(
+        &self,
+        port: u16,
+        additional_ssh_args: Option<&Vec<&str>>,
```

Comment:
I think option isnt necessary since it can just be empty

Reply:
Hm, so usually Vec does not need Option, right?